templates/agent-conf.d/kubernetes_state.yaml.erb

```diff
@@ -0,0 +1,22 @@
+init_config:
```

Comment:
`kubernetes_state` is a different integration, if you want to add this you'll need to have a separate manifest for it: `manifests/integrations/kubernetes_state.pp`

Reply:
Ah maybe I miss understood @irabinovitch request to not forget kubernetes_state_metrics. I reviewed the dd docs, but nothing states what is added to the yaml. Only the kubernetes pods file.

Reply:
I have added the tests, but these are my first spec tests ever. So please let me know if I miss understood anything or what to rework.